docs/install.md

```diff
@@ -22,6 +22,7 @@ This guide explains how to install Tekton Pipelines.
 ## Prerequisites
 
 -   A [Kubernetes cluster][k8s] running version 1.23 or later.
+-   If you are running on `macOS`, make sure Docker is running
```

Comment:
Why only when running `macOS`? Why Docker?
This is a list of prerequisites for installing Tekton on Kubernetes - the first in the list is a Kubernetes cluster - the procedure is not really specific to where that kubernetes cluster is running.